pkg/ip/Dockerfile

```diff
@@ -16,4 +16,5 @@ ENTRYPOINT []
 CMD []
 WORKDIR /
 COPY --from=mirror /out/ /
+COPY linuxkit-configure-wireguard-iface.sh /usr/bin/wg-config
```

Comment:
Please don't name this binary `wg-config`, as that's a name previously used and possibly used in the future by upstream. Something linuxkit-specific might be better, such as `linuxkit-configure-wireguard-iface`. Alternatively, maybe you prefer a short linuxkit prefix -- `lk-wg-config`?